common/utils/src/main/resources/error/error-conditions.json

```
@@ -3521,6 +3521,12 @@
     ],
     "sqlState" : "0A000"
   },
+  "NEGATIVE_VALUES_IN_FREQUENCY_EXPRESSION" : {
+    "message" : [
+      "Negative values found in <frequencyExpression>"
```

Let's help users in trouble shooting, and show the value:
```suggestion
      "Found the negative value in <frequencyExpression>: <negativeValue>, but expected a positive integral value."
```
just pass the value at:
```scala
      } else if (frqLong < 0) {
        throw QueryExecutionErrors.negativeValueUnexpectedError(frequencyExpression)
      }
```

---

Sounds good. Just applied the comments. Thanks!